src/platform/silabs/efr32/BLEManagerImpl.cpp

```diff
@@ -1123,8 +1293,7 @@ extern "C" void sl_bt_on_event(sl_bt_msg_t * evt)
             chip::DeviceLayer::Internal::BLEMgrImpl().HandleTxConfirmationEvent(
                 evt->data.evt_gatt_server_characteristic_status.connection);
         }
-        else if ((evt->data.evt_gatt_server_characteristic_status.characteristic == gattdb_CHIPoBLEChar_Tx) &&
-                 (evt->data.evt_gatt_server_characteristic_status.status_flags == sl_bt_gatt_server_client_config))
+        else
```

Comment:
how do we let the User of the side channel handle other possible event they might be interested on.

With this implementation we don't let them handle any

Reply:
Do tou mean that the switch case statement should have a `default` that is handled by the side channel? If that is the case, I agree, I'll look into implementing a virtual method for handling other events in the BLEChannel object.

If this is regarding sl_bt_evt_gatt_server_characteristic_status_id, to my knowledge this is use either to recevie request for subscription to a characteristic supporting indication or to confirm indication was sent. The side channel implementation can decide what logic they want to use in the `HandleCCCDWriteRequest`